buildSrc/src/main/groovy/com/uber/okbuck/composer/android/AndroidBuckRuleComposer.groovy

```diff
@@ -34,7 +34,7 @@ abstract class AndroidBuckRuleComposer extends JvmBuckRuleComposer {
         return "app_lib_${target.name}"
     }
 
-    static String bin(AndroidAppTarget target) {
```

Comment:
Having `bin_` for android libraries does not make sense. Lets name these differently since the `bin_` is supposed to be for actual apps.